src/AEOS/AEOSConnector/Connectors/AEOSConnector.cs

```diff
@@ -0,0 +1,227 @@
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Serilog;
+
+namespace Innovatrics.SmartFace.Integrations.AEOSConnector.Connectors
+{
+    public class AEpuConnector : IAEOSConnector
```

Comment:
class should not be AEOSConnector when the inferface is IAEOSConnector as well ?

Reply:
It was renamed to AEpuConnector, the whole projects naming was refactored.